atcodertools/common/language.py

```diff
@@ -61,7 +61,7 @@ def from_name(cls, name: str):
     name="java",
     display_name="Java",
     extension="java",
-    submission_lang_pattern=re.compile(".*Java8.*"),
+    submission_lang_pattern=re.compile(".*Java.*"),
```

Comment:
古いジャッジにJava7がいてそれがマッチしたら悲しいのでこうしましょう
```suggestion
    submission_lang_pattern=re.compile(".*Java8.*|.*Java \\(.*"),
```